user_pinger.py

```diff
@@ -448,6 +448,33 @@ def list_groups(_, author: praw.models.Redditor) -> None:
             self._send_pm("Avaliable Groups", [groups_list], author)
             return
 
+        def list_my_groups(_, author: praw.models.Redditor) -> None:
+            """
+            Returns all the groups the user is a part of
+
+            Usage:
+            body = listmygroups
+            """
+            self.logger.debug("Getting groups")
+            groups: ConfigParser = self._get_wiki_page(["config", "groups"])
+            self.logger.debug("Got groups")
+
+            self.logger.debug("Getting User's groups")
+            myGroups = []
+
+            for (group_name, usernames) in groups.items():
+                    for username in usernames.items():
+                        username = str(username[0]).upper()
+                    if str(author) == username.upper():
```

Comment:
username.upper() still redundant.

also I think the indent is wrong here, pretty sure the if should be inside the for loop?